### DISCUSSION TITLE
WT32-ETH Issue with WiFi.h

### DISCUSSION THREAD

**Discussion by mauricemullens** [eModbus/eModbus#366] [eModbus applications]
I just upgraded to 1.7.2 from 1.7.0 and found the following issue:
/.platformio/packages/framework-arduinoespressif32/libraries/Ethernet/src/ETH.h:27:10: fatal error: WiFi.h: No such file or directory
I tried reinstalling the platformio and update all libs, but to no avail.
Any idea what could be wrong?

I was using #include "ModbusServerETH.h" on 1.7.0, all working fine.
Thanks!
Maurice

**Comment by mauricemullens** [comment]
lib_ldf_mode = deep+ is the culprit here. When i remove it, it builds

**Reply by Miq1** [reply]
Astonishing! That ``deep+`` is necessary with my setup to just avoid the issue you are describing. I have no idea what may cause it - the library management inside PlatformIO must behave differently.

**Reply by mauricemullens** [reply]
i guess so .. well, if it works it works :-)
I do have some other questions i run into in my current use of your library, which i greatly greatly appreciate! I will put it in a different discussion (topic). Thanks!
